tests/unit/type/stream-cgroups.tcl

```diff
@@ -786,6 +786,41 @@ start_server {
         }
     }
 
+    start_server {tags {"external:skip"}} {
+        set master [srv -1 client]
+        set master_host [srv -1 host]
+        set master_port [srv -1 port]
+        set slave [srv 0 client]
+
+        foreach autoclaim {0 1} {
+            test "Replication tests of XCLAIM with deleted entries (autclaim=$autoclaim)" {
+                $slave slaveof $master_host $master_port
```

Comment:
i know you copied it from somewhere, but let's avoid the salve term in new code, so that we don't have to destroy the blame log of that line if we ever decide to rename all uses of it.